guides/v2.1/howdoi/checkout/checkout_carrier.md

```diff
@@ -53,6 +53,9 @@ define(
                     },
                     'country_id': {
                         'required': true
+                    },
+                    'city': {
```

Comment:
Hi @magesanjay 
Can you please add city field in Line no:39 Like  **Country**, **Zip Code** and **City**

Reply:
Hello @ronak2ram
I have updated content also as per your request.

Reply:
@magesanjay thanks for the update PR. PR looks good now.